.gitignore

```diff
@@ -4,3 +4,4 @@
 *.log
 build/
 dist/
+.idea/
```

Comment:
@0lha Please remove this.
Non related to application ignores should be added to you local git configuration instead of project global .gitignore file.

Reply:
done